### PR TITLE
CosVM Network

### PR DESCRIPTION
**Pull request by Financiya-Network**
Please Add Chain

### CODE REVIEW

**Review by D777del27 — APPROVED** [review]

**Comment by cosvmlabs** [comment]
Please Merge

**Review by cosvmlabs — APPROVED** [review]
approved

**Comment by cosvmlabs** [comment]
Please see This Change

File: _data/chains/eip155-323.json
Comment (cosvmlabs) [inline _data/chains/eip155-323.json:1]: Chk Please

**Review by cosvmlabs — APPROVED** [review]

File: _data/chains/eip155-323.json
Comment (vehicles4real) [reply _data/chains/eip155-323.json:1]: Cosmos

**Review by cosvmlabs — APPROVED** [review]

**Comment by cosvmlabs** [comment]
Please Add in list

File: _data/chains/eip155-323.json
Comment (cosvmlabs) [reply _data/chains/eip155-323.json:1]: not understand

**Comment by cosvmlabs** [comment]
image

**Pull request closed without merging**